### DISCUSSION TITLE
How to detect if two connectors are compatible?

### DISCUSSION THREAD

**Discussion by lucian-torje-siemens** [eclipse-edc/.github#38] [Q&A]
Hello,
How can we detect if two connectors are compatible upfront (before trying to do a data transfer for example)?

I'm asking this since in an environment with hundreds or connectors, if some connectors are not compatible  and we could know it upfront (e.g. if one connector is missing httpdata support and the other needs it), this would allow us to lower the maintenance costs (less preventable investigations).

**Comment by mspiekermann** [comment]
Compatibility mainly depends on supported versions of the [Dataspace Protocol](https://projects.eclipse.org/projects/technology.dataspace-protocol-base).
